src/icon-manifest/manifestMerger.ts

```diff
@@ -87,7 +87,7 @@ export function toggleAngularPreset(
   disable: boolean,
   files: models.IFileCollection): models.IFileCollection {
   const icons = files.supported
-    .filter(x => x.icon.startsWith('ng_') && (x.icon.endsWith('_js') || x.icon.endsWith('_ts')))
+    .filter(x => x.icon.startsWith('ng_') && !x.icon.endsWith('2'))
```

Comment:
1. You have missed the `scss` extension.
2. The second part of the condition should better check against `number`. We could have in the future a 3, 4, 5... version.